src/cmd/cli/command/commands.go

```diff
@@ -89,8 +89,14 @@ func Execute(ctx context.Context) error {
 			projectName := "<name>"
```

Comment:
Let's move this if into `makeComposeUpCmd` https://github.com/DefangLabs/defang/blob/b3451afbf239738c6ae1a71b55feb004c8ac69fa/src/cmd/cli/command/compose.go#L41

Reply:
ok, it's updated now

Reply:
ok, it's updated now